tests/network/bgp/evpn/conftest.py

```
@@ -0,0 +1,197 @@
+    cudn_evpn_route_advertisements: None,
+    frr_configuration_evpn: None,
+    workers: list[Node],
+) -> None:
+    node_names = [worker.name for worker in workers if worker.name != frr_external_pod.pod.instance.spec.nodeName]
+    wait_for_evpn_established(frr_pod=frr_external_pod.pod, expected_neighbors=len(node_names))
+
+
+@pytest.fixture(scope="module")
+def vm_evpn_target(
+    namespace_evpn: Namespace,
+    cudn_evpn_layer2: libcudn.ClusterUserDefinedNetwork,
```

Ruff (0.15.10)

[warning] Unused function argument: `cudn_evpn_layer2`

(ARG001)